ignite/metrics/classification_report.py

```diff
@@ -4,7 +4,6 @@
 import torch
```

Comment:
Let's keep classification report updates to another PR

Reply:
Its tests woulf fail, then. Shall I mark them as `xfail`?

Reply:
This is curious why the tests are failing then. As right now tests are passing and we introducing new features trying to keep BC, I would expect that tests won't break...

Reply:
You're right. They had been marked as `xfail` before.